### DISCUSSION TITLE
Where can I learn and upgrade my knowledge about python?

### DISCUSSION THREAD

**Discussion by s0VEstb** [codewars/codewars.com#3134] [General]
can you help me? im going to learn telegram bots and Django framework, should I learn CSS and HTML, or learn DataBase (SQL)

**Comment by hobovsky** [comment]
Codewars Github is not meant for help with general programming questions. It's meant to discuss matters related to Codewars platform, the website, its functions, etc.

For help with your questions please visit your favorite general programming forum, a general Python forum, or you can try your luck asking for help on Codewars Discord.

Closing as off-topic.
